src/fs/dir.rs

```diff
@@ -111,6 +111,13 @@ impl<'dir, 'ig> Files<'dir, 'ig> {
                     continue;
                 }
 
+                // Also hide _prefix files on Windows because it's used by old applications
+                // as an alternative to dot-prefix files.
+                #[cfg(windows)]
+                if ! self.dotfiles && filename.starts_with('_') {
```

Comment:
I think this is overdoing it. Python uses `__init__.py` for top-level modules, and this hides all such files. If you're running this build with msys/cygwin, most likely your dotfiles actually start with '.' anyway.

Reply:
I don't know any files, except for `_vimrc` on Windows that follows this convention either.

Reply:
> If you're running this build with msys/cygwin, most likely your dotfiles actually start with '.' anyway

Even without msys/cygwin. I don't mess with those and only use pwsh, and just use the normal dot prefix for all my dotfiles.